src-tool/Pact/Analyze/Check.hs

```diff
@@ -389,7 +391,7 @@ verifyFunctionInvariants
   -> Text
   -> CheckableType
   -> IO (Either CheckFailure (TableMap [CheckResult]))
-verifyFunctionInvariants (CheckEnv tables _consts _pDefs moduleData caps gov _de)
+verifyFunctionInvariants (CheckEnv  tables _consts _pDefs moduleData caps gov _de mDebug)
```

Comment:
Just in the name of consistency, having two spaces after a constructor name is not something you find elsewhere...